server/tserver/src/main/java/org/apache/accumulo/tserver/UnloadTabletHandler.java

```diff
@@ -114,19 +114,14 @@ public void run() {
     try {
       TServerInstance instance =
           new TServerInstance(server.clientAddress, server.getLock().getSessionId());
-      TabletLocationState tls = null;
-      try {
-        tls = new TabletLocationState(extent, null, Location.current(instance), null, null, null,
-            false, TabletHostingGoal.ONDEMAND, false);
-      } catch (BadLocationStateException e) {
-        log.error("Unexpected error", e);
-      }
+      TabletMetadata tm = new TabletMetadataImposter(extent, null, Location.current(instance), null,
```

Comment:
We may want to open up a follow on issue here and add a TODO.  Eventually the tablet may have a cahced version of a tabletmetadata obj it could pass here.  Not sure though.

Reply:
TODO added in 9153726